tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_login_bzlite.py

```diff
@@ -1,17 +0,0 @@
-from gaiatest import GaiaTestCase
```

Comment:
If we merge your PR as is, we won't run this test anymore. You have to take out the deletion of the file from this PR.

```
git checkout HEAD^ tests/python/gaia-ui-tests/gaiatest/tests/functional/bzlite/test_login_bzlite.py
git commit
```